docs/dev/core/dependency-vetting.md

```diff
@@ -0,0 +1,62 @@
+# Dependency vetting
+
+[cargo-vet]: https://mozilla.github.io/cargo-vet/index.html
+
+The Glean SDK uses `cargo-vet` to ensure that third-party Rust dependencies have been audited by a trusted entity.
+For a full overview over `cargo-vet`'s capabilities and usage see the [`cargo-vet` documentation][cargo-vet].
+
+New or updated dependencies need to be audited to allow their usage.
+Dependency audits are shared with downstream Mozilla projects.
+
+> **Note:** Please be thorough when auditing dependencies.
```

Comment:
What does "be thorough" mean?

Reply:
actually this was previously coupled with the sentence above about sharing it with mozilla.
I think calling it out there is enough.